src/lib/AccountStore.js

```diff
@@ -96,6 +98,7 @@ class AccountStore {
      */
     async dangerousListPlain() {
         const db = await this.connect();
+        if (!db) return [];
```

Comment:
same here. I always find it weird/ugly if a method returns a promise sometimes and an object other times. :/

Reply:
So my suggestion is to move this `if` on line down and go `if (!db) resolve([]);`

Reply:
But it's an `async` function, so it returns a promise anyway, no?

Reply:
yes, it returns a promise anyway and thus I'd say `return [];` is easier to read